WebDevStudios/nextjs-wordpress-starter
# Adds Description Prop to Gravity Form Component

Closes #

### Description

When working with GravityForm, I notice that the form description is not been pass to the component, I add the description prop to it.

### Screenshot

![Screen Shot 2021-04-27 at 10 36 10](https://user-images.githubusercontent.com/3707088/116279137-6cce3480-a744-11eb-95b6-eab12065cc17.png)


### Verification

How will a stakeholder test this?

1. Add a Description to a GravityForm on your WP install
2. Save It
3. Open the page on NextJS, see the description under the title
